### DISCUSSION TITLE
So here's why it hangs if you don't use C

### DISCUSSION THREAD

**Discussion by v1993** [tallbl0nde/TriPlayer#52] [General]
https://github.com/tallbl0nde/TriPlayer/blob/737adfdfcea89924286b1cd13ec47a9ebaffda96/Sysmodule/source/main.cpp#L14-L25

Came across this comment and couldn't resist, sorry: it's most likely [name mangling of C++](https://en.wikipedia.org/wiki/Name_mangling#C++) - functions end up with names different from what they are in code, making loader unable to locate corresponding symbols and fail to load sysmodule.

**Comment by tallbl0nde** [comment]
That... makes a lot of sense. Thanks for clearing that up! :)
(I don't even remember writing that comment!)
